src/controllers/livetv/livetvsuggested.js

```diff
@@ -1,395 +1,403 @@
-                    }
-                }
+    function loadTab(page, index) {
+        currentTabIndex = index;
+        getTabController(page, index, function (controller) {
+            initialTabIndex = null;
 
-                callback(controller);
-            });
-        }
-
-        function preLoadTab(page, index) {
-            getTabController(page, index, function (controller) {
-                if (renderedTabs.indexOf(index) === -1 && controller.preRender) {
-                    controller.preRender();
+            if (renderedTabs.indexOf(index) === -1) {
+                if (index === 1) {
+                    renderedTabs.push(index);
                 }
-            });
-        }
-
-        function loadTab(page, index) {
-            currentTabIndex = index;
-            getTabController(page, index, function (controller) {
-                initialTabIndex = null;
 
-                if (renderedTabs.indexOf(index) == -1) {
-                    if (index === 1) {
-                        renderedTabs.push(index);
-                    }
+                controller.renderTab();
+            } else if (controller.onShow) {
+                controller.onShow();
+            }
 
-                    controller.renderTab();
-                } else if (controller.onShow) {
-                    controller.onShow();
-                }
+            currentTabController = controller;
+        });
+    }
 
-                currentTabController = controller;
-            });
+    function onInputCommand(evt) {
+        if (evt.detail.command === 'search') {
+            evt.preventDefault();
+            Dashboard.navigate('search.html?collectionType=livetv');
         }
+    }
 
-        function onInputCommand(evt) {
-            if (evt.detail.command === 'search') {
-                evt.preventDefault();
-                Dashboard.navigate('search.html?collectionType=livetv');
-            }
-        }
+    let isViewRestored;
+    const self = this;
+    let currentTabIndex = parseInt(params.tab || getDefaultTabIndex('livetv'));
+    let initialTabIndex = currentTabIndex;
+    let lastFullRender = 0;
+    [].forEach.call(view.querySelectorAll('.sectionTitleTextButton-programs'), function (link) {
+        const href = link.href;
 
-        var isViewRestored;
-        var self = this;
-        var currentTabIndex = parseInt(params.tab || getDefaultTabIndex('livetv'));
-        var initialTabIndex = currentTabIndex;
-        var lastFullRender = 0;
-        [].forEach.call(view.querySelectorAll('.sectionTitleTextButton-programs'), function (link) {
-            var href = link.href;
+        if (href) {
+            link.href = href + '&serverId=' + ApiClient.serverId();
+        }
+    });
 
-            if (href) {
-                link.href = href + '&serverId=' + ApiClient.serverId();
-            }
-        });
+    self.initTab = function () {
+        const tabContent = view.querySelector('.pageTabContent[data-index=\'0\']');
```

Comment:
```suggestion
        const tabContent = view.querySelector('.pageTabContent[data-index="0"]');
```

The rules allow for double quotes in order to avoid escaping, but in this case, the usual quote to use for selectors is double quotes.